src/task-definitions.js

```diff
@@ -266,6 +248,14 @@ tasks.exportNotebook = new UserTask({
   callback() { dispatcher.exportNotebook() },
 })
 
+tasks.exportNotebookAsReport = new UserTask({
+  title: 'Export Notebook as report',
+  keybindings: ['shift+e'],
+  displayKeybinding: 'Shift E',
```

Comment:
I know this is extremely ugly right now, but could we replace `Shift` with `\u21E7 `? I need to do a small refactor that automatically generates the right human-readable keybinding, but for now this works on the other menu items that use `Shift`.

Reply:
Hmmm I'm looking at some mac keyboards right now and they all label that key with the text "shift". I've looked at a couple other cross platform programs, and the convention seems to be to write out the word rather than use a symbol. If using a symbol is the convention on mac, we can a add that later but I'm in favor of keeping the explicit string as the base case for other platforms (and for mac until Mike's suggesting is implemented, since "shift" will at least not be confusing)

Reply:
hmm, i'm going back and looking at the menus, and i'm actually finding the use of symbols in general to be somewhat ambiguous, esp in the case of the up and down arrows, where there are a couple arrows next to each other.

![image](https://user-images.githubusercontent.com/2380975/37247045-f02b4392-2469-11e8-877e-1728e8b2009a.png)

I just went to go look at the FF menus on my girlfriend's Mac, and i see that using these symbols is the standard there... but she didn't even know what all of them meant. With the exception of the "open apple" symbol (which is unambiguous on a mac kbd) and *maybe* the shift arrow, I don't think we should be using symbols for the kbd shorcuts, and definitely not as the default for all platforms. If we do use them it should be as a variant for mac only.

As long as i'm in here, i'm going to change these default displays as well. I'll leave the mac symbols in the comments so that we won't have to hunt them down once we're ready to implement the mac variants.